### ISSUE TITLE
Error triggered by fibonacci prompt (but not by factorial!)

### ISSUE THREAD

**Issue by sciabarracom** [microsoft/genaiscript#848]
If I ask to generate a fibonacci function with this prompt:

```
Generate the code for a fibonacci function in python.
```

I always get this error:

```
Cannot read properties of undefined (reading 'tagName')
TypeError: Cannot read properties of undefined (reading 'tagName')
    at OrderedObjParser.parseXml (/home/ubuntu/mastrogpt2/node_modules/genaiscript/built/genaiscript.cjs:9797:34)
    at XMLParser2.parse (/home/ubuntu/mastrogpt2/node_modules/genaiscript/built/genaiscript.cjs:10182:48)
    at XMLParse (/home/ubuntu/mastrogpt2/node_modules/genaiscript/built/genaiscript.cjs:58646:17)
    at XMLTryParse (/home/ubuntu/mastrogpt2/node_modules/genaiscript/built/genaiscript.cjs:58624:12)
    at tracePromptResult (/home/ubuntu/mastrogpt2/node_modules/genaiscript/built/genaiscript.cjs:94252:51)
    at runTemplate (/home/ubuntu/mastrogpt2/node_modules/genaiscript/built/genaiscript.cjs:94484:5)
    at async runScript (/home/ubuntu/mastrogpt2/node_modules/genaiscript/built/genaiscript.cjs:95395:14)
    at async _Command.runScriptWithExitCode (/home/ubuntu/mastrogpt2/node_modules/genaiscript/built/genaiscript.cjs:95266:1
```

The command line is:
``` 
npx genaiscript run code --model github:Meta-Llama-3.1-70B-Instruct  spec/code/fib.txt 
```
and the script is
```
def("FILE", env.files)
$`Generate the code described in FILE`
```

Note that if I use as a prompt `Generate the code for a factorial function in python.`  then there are no errors.

It is puzzling why the error is triggered by the fibonacci and not by the factorial.
The error happens also with other models (github:gpt-4o for example).

**Comment by pelikhan** [comment]
could you run `npx genaiscript --version` ? This should be fixed in 1.73.0. Or try forcing the update

```
npx genaiscript@^1.73.0 run code --model github:Meta-Llama-3.1-70B-Instruct  spec/code/fib.txt
```

**Comment by sciabarracom** [comment]
Confirmed. It works with 1.74.1
